flaskbb/templates/forum/topic.html

```diff
@@ -133,6 +146,34 @@
                                 <a href="{{ url_for('forum.report_post', post_id=post.id) }}" onclick="window.open(this.href, 'wio_window','width=500,height=500'); return false;" class="btn btn-icon icon-report" data-toggle="tooltip" data-placement="top" title="Report this post"></a>
                             {% endif %}
 
+                            {% if current_user.permissions.get('makehidden') %}
```

Comment:
Extremely unhappy with this, any ideas?

Reply:
Mhh, it's probably not related to this PR but we could move the post controls to a seperate file like we did with the topic controls.